ground-control/internal/server/group_handlers.go

```
@@ -0,0 +1,136 @@
+	if err != nil {
+		log.Println(err)
+		HandleAppError(w, err)
+		return
+	}
+	// Create a new Queries object bound to the transaction
+	q := s.dbQueries.WithTx(tx)
+	// Ensure proper transaction handling with defer
+	defer func() {
+		if p := recover(); p != nil {
+			// If there's a panic, rollback the transaction
+			tx.Rollback()
```

golangci-lint (1.64.8)

Error return value of `tx.Rollback` is not checked

(errcheck)